Source/cairo/CairoContext.m

```diff
@@ -70,6 +70,12 @@
 #    define _CAIRO_SURFACE_CLASSNAME Win32CairoSurface
 #    include "cairo/Win32CairoSurface.h"
 #  endif /* USE_GLITZ */
+#elif BUILD_SERVER == SERVER_wayland
+#  include "wayland/WaylandServer.h"
+#  define _CAIRO_GSTATE_CLASSNAME CairoGState
+#  include "cairo/CairoGState.h"
```

Comment:
i would move the includes before the defines.

Reply:
Done. FWIW this is not the case in other `#if`s.